src/metabase/util/number/impl.clj

```diff
@@ -1,7 +1,7 @@
 (ns metabase.util.number.impl
   "CLJ implementation of the number utilities.
   See [[metabase.util.number]] for the public interface."
-  (:refer-clojure :exclude [bigint])
+  (:refer-clojure :exclude [bigint integer?])
```

Comment:
It doesn't need to be part of this PR, but I think for files like these that don't contain any macros etc., it would probably be easier to read if they were a single `.cljc` file with `#?(:clj ... :cljs ...)` inside.

Otherwise it's a headache to keep them in sync.

Reply:
I'll change that in a separate PR